dynamics/apis/api_base.py

```diff
@@ -75,8 +75,13 @@ def _get_request(self, params, api_url):
             headers=api_headers,
             params=api_params
         )
-
+        
         if response.status_code == 200 or response.status_code == 201:
+            raw_content = response.content
```

Comment:
remove changes from these lines since there is a new method for count

Reply:
please remove all the changes from this get request

test it out well
Both get call as well get count call

Reply:
done